cncf-youtube-channel-summarizer/README.md

```diff
@@ -0,0 +1,28 @@
+# CNCF Youtube Channel Video Summarizer
+
+## Project Goals
+* Use Large Language Model (LLM) to summarize, categorize videos from CNCF YouTube channel.
```

Comment:
Thanks for the PR. 

In the discussion, we identified two goals: one is AI for Cloud Native (i.e. your writing here) and the other is Cloud Native for AI.

Would you please add a line for the second goal, e.g. demonstrate how to use Cloud Native infrastructure to serve an AI application like this one? This is a stretch goal, depending on resource availability.